src/backend/vulkan/src/command.rs

```diff
@@ -206,7 +206,7 @@ impl command::RawCommandBuffer<Backend> for CommandBuffer {
                         region.src_subresource,
                         region.num_layers,
                     ),
-                    src_offset: offset,
+                    src_offset: offset.clone(),
```

Comment:
Ugh, `vkOffset3d` not implementing `Copy`.
We should address this in `ash` in the future.